CONTRIBUTING.rst

```diff
@@ -209,6 +214,15 @@
    cmake path/to/fish-shell
    make test
 
+Or you can run them on a fish, without involving cmake::
+
+  cargo build
+  FISHDIR=target/debug tests/test_driver.sh tests/test.fish # script tests, the checks
+  FISHDIR=target/debug tests/test_driver.sh tests/interactive.fish # interactive tests, the pexpects
```

Comment:
in future we want to re-add some ctest niceties, such as printing a list of all failed test at the end